### PR TITLE
updated version 0.3.0 with new outgoing actions

### PR DESCRIPTION
**Pull request by jyothishnt**
https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-1591

Updating version to 0.3.0

Added OutgoingActions for turning on/off several transcripts

Also created a pull request to test this: https://github.com/Ensembl/ensembl-client/pull/759

### CODE REVIEW

File: src/types.ts [hunk 1]
```diff
@@ -209,6 +211,20 @@ export type TurnOffNamesAction = {
   };
 };
 
+export type TurnOnSeveralTranscriptsAction = {
+  type: OutgoingActionType.TURN_ON_SEVERAL_TRANSCRIPTS;
+  payload: {
+    track_ids: string[];
```
Comment (azangru) [inline src/types.ts:217]: Here and below — why this payload? You aren't using it when sending the message to genome browser.

**Comment by azangru** [comment]
Before merging this in, we'll have to see it in action in an ensembl-client review deployment

File: src/types.ts (hunk 1 above)
Comment (jyothishnt) [reply src/types.ts:217]: true. that was a copy paste mistake. What do we send for turning on this?

Comment (azangru) [reply src/types.ts:217]: Nothing; from your updates to send.ts, the type should be sufficient.

Comment (azangru) [reply src/types.ts:217]: We reviewed my comment at a daily scrum; and I realised I was wrong. I misread track ids as transcript ids. Please ignore my comment.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ensembl/ensembl-genome-browser",
-  "version": "0.2.1",
+  "version": "0.3.0-test",
```
Comment (imransl) [inline package.json]: **NOTE TO SELF**: Update this to `0.3.0` before merging

**Review by azangru — APPROVED** [review]
Looks good

**Pull request merged**